text/0000-fun-vs-fun-ptr.md

```diff
@@ -0,0 +1,104 @@
+- Feature Name: explicit_fun_ptr
+- Start Date: 2015-2-17
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+It is likely that Rust will want to someday have a notion of an unboxed function type to be used
+with custom pointer types[1]. Ideally then today's function pointers would become `&'static`s of
```

Comment:
Oh right. There's usually a way to do superscripts but I think it's nonstandard, `foo^1` on reddit and `foo<sup>1</sup>` on GH. (let's<sup>see...</sup>)